src/api/src/services/LogFile/index.ts

```diff
@@ -0,0 +1,19 @@
+import { Service } from 'typedi';
+import readLastLines from 'read-last-lines';
+
+import LogFileArgs from '../../graphql/args/LogFile';
```

Comment:
It would be better not to include transport layer args file in the service layer. I know that currently this rule is violated in some parts of the project, but we are trying to steer away from it.

As an alternative you could define args using interface in the current service.